lib/autoload/kuroko2/workflow/task/fork.rb

```diff
@@ -32,6 +31,19 @@ def validate
 
         private
 
+        def children
+          if fork_children_ids.present?
```

Comment:
this should be always Array; empty? is enough. `present?` doesn't perform a empty-ness check only.